projects/go-lib/src/lib/components/go-pill/go-pill.component.scss

```diff
@@ -0,0 +1,28 @@
+@import '../../../styles/variables';
+
+.go-pill {
+  background-color: $ui-color-primary;
+  border-radius: $global-radius;
+  color: $base-light;
+  cursor: default;
+  display: inline-flex;
+  font-size: 0.75rem;
+  font-weight: $weight-regular;
+  margin-bottom: 0.5rem;
```

Comment:
Only if it makes sense semantically. Is the reason this value needs to be `0.5rem` because we want it to be half the size of a column gutter, or is that coincidental?

Reply:
I updated two of the margins to use use the variable, but the other two are not really coupled to the gutter size (meaning they will stay .5rem even if the gutter size were to change), so I left them hard coded.